apps/hannk/halide/depthwise_conv_generator.cpp

```diff
@@ -8,8 +8,8 @@ namespace hannk {
 
 class DepthwiseConv : public Generator<DepthwiseConv> {
 public:
-    // If positive, a constant inverse depth multiplier.
-    GeneratorParam<int> inv_depth_multiplier_{"inv_depth_multiplier", -1};
+    // This is used to compute ci = co * inv_depth_multiplier
```

Comment:
Is it only to support broadcasting version, when inv_depth_multiplier is 0? If yes, it'd be helpful to put this into the comment.